.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi" : true,
+    "singleQuote": true
+}
```

Comment:
please add a blank line at the end of this file. Github will complain if there is not one

Reply:
blank line added